src/data.ts

```diff
@@ -4,6 +4,7 @@ export const defaultSocialiteNetworks = [
   networks.facebook,
   networks.instagram,
   networks.linkedin,
+  networks.onlyfans,
```

Comment:
I probably wouldn't consider this a "default" and might remove it from the `defaultSocialiteNetworks` export later.